soccer/KickEvaluator.cpp

```diff
@@ -315,7 +315,7 @@ tuple<float, float> KickEvaluator::rect_to_polar(const Point origin,
 
 vector<tuple<float, float> > KickEvaluator::convert_robots_to_polar(
     const Point origin, const Point target) {
-    vector<Robot*> bots = get_valid_robots();
+    vector<Robot*> bots = move(get_valid_robots());
```

Comment:
function return values are already temporary values so the move won't really do anything here.  move just casts things to an rvalue and function returns are rvalue.